bridge/client/app/_utils/date.utils.ts

```diff
@@ -41,4 +42,8 @@ export default class DateUtil {
       sameElse : 'YYYY-MM-DD HH:mm'
     };
   }
+
+  static compareTime(a: Trace, b: Trace) {
```

Comment:
[minor suggestion] do you think we should change the name to `compareTraceTimes` or add some documentation that it's clear that not actual timestamps are compared but Trace's `time` properties?

Reply:
Yeah, makes sense to rename the function to `compareTraceTimes`. Fixed it.